services/ocr/main.py

```diff
@@ -20,12 +20,11 @@ def callback(ch, method, properties, body):
         img = cv2.imdecode(nparr, cv2.IMREAD_COLOR)
 
         out = Engine.process(img, int(data["week_number"]), RABBITMQ_HOST, data["id"])
-        print(f"Sending data to scan:shift")
```

Comment:
Tohle jsi tam klidně mohl nechat

Reply:
Vzhledem k tomu, že tam posílám jen status DONE, tak jsem to radši odebral, aby to nebylo matoucí